lib/ipam/ipam_types.go

```diff
@@ -117,3 +117,39 @@ type PoolUtilization struct {
 	// Utilization for each of this pool's blocks.
 	Blocks []BlockUtilization
 }
+
+// Consumer represent a POD consuming an IP address.
```

Comment:
Removed any reference to POD. I am not sure though at this point how the table will look in case consumer is not a POD. Is there a way for me to simulate that?